kong/plugins/request-transformer/migrations/postgres.lua

```diff
@@ -0,0 +1,64 @@
+return {
+  {
+    name = "2016-05-04-160000_req_trans_schema_changes",
+    up = function(_, _, factory)
+
+      local plugins, err = factory.plugins:find_all {name = "request-transformer"}
+      if err then
+        return err
+      end
+
+      for _, plugin in ipairs(plugins) do
+        for _, action in ipairs {"remove", "add", "append", "replace"} do
```

Comment:
Would it be possible to change this line to include all 5 of the config options? Currently, it's missing rename. Would this work?
```for _, action in ipairs {"remove", "add", "append", "replace", "rename"} do```